test/rules/experiments/nnbd/rules/overridden_fields.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2016, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year

Reply:
We've been preserving the years for copies of existing tests.  Happy to rethink though?